### PR TITLE
Epic cactus94 pull request

### PR DESCRIPTION
**Pull request by EpicCactus94**
## Describe your project in 1-3 sentences

Creating a reaction wheel cube which is a cube that will be able to balance on one of its corners on level ground and on an inclined plane. The cube will be able to balance through 2 motors which will produce enough centrifugal force to keep the cube on one corner. It will also be able to react to disturbances such as a push. 

## Hack Club Slack handle
Ali Mustafa

## Checklist before submitting a proposal:

- [x] I am a current high school student (or younger).
- [x] I have submitted an RSVP at <https://hackclub.com/winter>
- [x] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [x] I have self-reviewed my project proposal.
[EpicCactus94.md.txt](https://github.com/hackclub/winter/files/10421147/EpicCactus94.md.txt)

### CODE REVIEW

**Comment by bellesea** [comment]
heyy thanks so much for making a PR! **we're back on tuesday and will start reviewing PRs in the order that they came in, thanks for your patience :)**

to make the process smoother, please ensure you double checked that your file is properly formatted, you have all the parts you need, and that your plan is detailed enough for us to imagine how you're using the components in your budget to build your project and visualise you work on it :)

and fyi, we're looking for projects where/that:
- you can spend at least 10 days building (from Feb 14th to 23rd)
- you are able to modify or make your own in some ways
- you can learn something new from
- involves working with electronics or software on a lower level than a pre-assembled kit of parts & ready software

and, we normally don’t approve projects where you buy consumer components w/o modification like keyboards (buying PCB, keycaps, switches, and assembling it), PCs (buying motherboard, cpu, memory and plugging it together), and kits (eg. closed-sourced consumer drone kits). 

_(note: this sounds soo awesome!! i'm wondering, are you referencing any sort of guides for this? also, does your budget include shipping and taxes?)_

our Jan 15 deadline is for PRs to come in and not for final approval so we'll work with you to either accept this project OR give you a chance to submit another PR. 

if you have any questions, please do not hesitate to ask her or DM me (i'm belle on slack), thank you!

File: EpicCactus94.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+name: "@EpicCactus94"
+project: "Self Balancing Reaction Wheel Cube"
+---
+
+# Project Name: Self Balancing Reaction Wheel Cube
+
+## Summary
+
+Design and 3D print a reaction wheel cube and code it to be able to balance on a corner (even on inclined surfaces) and react to disturbances 
+like pushing. I was inspired to build this project from videos online of how they're used in satelites.
```
Comment (exu3) [inline EpicCactus94.md:10]: Interesting! Could you link an example of such a video?

File: EpicCactus94.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+name: "@EpicCactus94"
+project: "Self Balancing Reaction Wheel Cube"
+---
+
+# Project Name: Self Balancing Reaction Wheel Cube
+
+## Summary
+
+Design and 3D print a reaction wheel cube and code it to be able to balance on a corner (even on inclined surfaces) and react to disturbances
```
Comment (exu3) [inline EpicCactus94.md:9]: What languages or frameworks might you use to program the wheel?

File: EpicCactus94.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+# Project Name: Self Balancing Reaction Wheel Cube
+
+## Summary
+
+Design and 3D print a reaction wheel cube and code it to be able to balance on a corner (even on inclined surfaces) and react to disturbances 
+like pushing. I was inspired to build this project from videos online of how they're used in satelites.
+
+## Plan
+
+I've already begun designing the .stl files which will be used to print the cube, and I've assured access to 3D printers before the building week.
```
Comment (exu3) [inline EpicCactus94.md:14]: Fascinating! Could you describe the process of creating these 3D models? What CAD tools are you using / what is your design process like?

File: EpicCactus94.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
+name: "@EpicCactus94"
```
Comment (exu3) [inline EpicCactus94.md:1]:
```suggestion
---
name: "@EpicCactus94"
```

**Review by exu3 — CHANGES_REQUESTED** [review]
Sounds cool! If you could add some more details to your proposal (see comments below) that would be great. It's looking good so far.

**Comment by EpicCactus94** [comment]
Hello. I apologize for my late response. It’s currently exam season at my
school so that’s why I’m a little behind on things. I’ll add a .md file
with all the updated information as soon as I can. Thank you for your
patience.

**Comment by exu3** [comment]
Ok! Whenever you're ready, just re-request a review. Good luck on exams.

**Comment by EpicCactus94** [comment]
Thanks for allowing me to still submit a proposal, even though it's late.
I've added more details to the .md file and it is now properly formatted.
I've created a new pull request and put all of the updated information on
there.

**Comment by exu3** [comment]
Closing in favor of #420

**Pull request closed without merging**